shared/cfg/guest-os/Linux/Fedora/28.cfg

```diff
@@ -93,7 +93,7 @@
                 md5sum_cd1 = 18740b445159c54d10bd887650e8d1d7
                 md5sum_1m_cd1 = cf06d7246d515d06ebb73e6e4a2f547f
             unattended_install.url:
-                url = http://dl.fedoraproject.org/pub/fedora/linux/releases/28/Server/${vm_arch_name}/os
+                url = https://archives.fedoraproject.org/pub/archive/fedora/linux/releases/28/Server/${vm_arch_name}/os
                 sha1sum_vmlinuz = 93b2351535ed8cfe9e3440b4bba2402d
```

Comment:
I see md5sum values being updated in place of sha1sum for all(archs) vmlinuz and initrd, not the new change, but better to correct now...

E:g:-
```
# md5sum vmlinuz
93b2351535ed8cfe9e3440b4bba2402d  vmlinuz

sha1sum suppose to be
# sha1sum vmlinuz
238e083e114c48200f80d889f7e32eeb2793e02a  vmlinuz
```

Reply:
sure, I have  updated the commit